### PR TITLE
Install bash completions files for adb and fastboot

### LINKED ISSUE #18: Install adb/fastboot bash completions
Upstream has bash completions for adb/fastboot

https://android.googlesource.com/platform/packages/modules/adb/+/refs/heads/master/adb.bash
https://android.googlesource.com/platform/system/core/+/refs/heads/master/fastboot/fastboot.bash

It would be great if cmake installs these files

### PR DESCRIPTION
**Pull request by nmeum**
Fixes #18

### CODE REVIEW

File: vendor/CMakeLists.txt [hunk 1]
```diff
@@ -61,3 +61,8 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_FILE_OFFSET_BITS=64")
 # Targets which should be installed by `make install`.
 install(TARGETS adb fastboot "${ANDROID_MKE2FS_NAME}"
 	simg2img img2simg append2simg DESTINATION bin)
+
+# Install bash completion files.
+set(BASH_COMPLETION "${CMAKE_INSTALL_PREFIX}/share/bash-completion/completions")
+install(FILES core/adb/adb.bash RENAME _adb DESTINATION "${BASH_COMPLETION}")
```
Comment (anatol) [inline vendor/CMakeLists.txt:67]: the source path is `adb/adb.bash` not `core/adb/adb.bash`.

Comment (anatol) [reply vendor/CMakeLists.txt:67]: also out of curiosity, why the installed filenames start with underscore? like `_adb` and `_fastboot`.

Comment (nmeum) [reply vendor/CMakeLists.txt:67]: > the source path is `adb/adb.bash` not `core/adb/adb.bash`.

Hm?

```
$ git log --format=oneline | head -1
c005b9c54a091f17c525c9e7a65d5afd36d63afc Add patch for missing <limits> include
$ find ./vendor -name adb.bash
./vendor/core/adb/adb.bash
```

> also out of curiosity, why the installed filenames start with underscore?

Whoops, confused that with zsh completion were completions need to be prefixed with an underscore for some reason. Will fix that later.

Comment (anatol) [reply vendor/CMakeLists.txt:67]: > ```
> $ git log --format=oneline | head -1
> c005b9c54a091f17c525c9e7a65d5afd36d63afc Add patch for missing <limits> include
> $ find ./vendor -name adb.bash
> ./vendor/core/adb/adb.bash
> ```

you are right.

I was testing it with 31.0.0 release and upstream has moved it to the new location https://android.googlesource.com/platform/packages/modules/adb.git/+/refs/tags/platform-tools-31.0.0

**Comment by nmeum** [comment]
Cherry picked with local modifications 45e337f2bf84ceb55c5625aafc5ab5793a91c023

**Pull request closed without merging**